### ISSUE TITLE
Export of Live Photo with --download-missing leads to a movie file with jpg extension

### ISSUE THREAD

**Issue by mwort** [RhetTbull/osxphotos#64]
If you export a live photo with the `--download-missing` flag, you end up with a seemingly corrupt file with a jpeg/JPG extension. exiftool will tell you it's a mov file and if you change the extension you see the movie file of the Live Photo. The actual photo is missing. It works as expected with the `--export-live` flag though (IMHO `--export-live` should be the default as it is with Photos as well).

**Comment by RhetTbull** [comment]
I found the problem but likely won't have time to implement a fix until the weekend.  The issue is in utils.py _export_photo_uuid_applescript:
https://github.com/RhetTbull/osxphotos/blob/1e013b6802e49e26ec5a94eb702e841b2eb68395/osxphotos/utils.py#L367-L372

I assumed Photos only exported a single item so took the first item from the glob (see comment in line 369!)  Indeed, if exporting a live photo with "unmodified originals", Photos actually exports 3 items: a jpeg, .mov, and a .xmp for the .mov (if "Export IPTC as XMP" is checked).

I'll add logic to check to see if we're exporting a live photo and if so, find the jpeg and .mov files and correctly export those.  The glob is needed because there's no way in applescript to get the name(s) of the files that Photos exported.  It's a very ugly interface but until I have time to figure out how to do this with PhotoKit, it's the only work-around for photos that aren't yet downloaded from iCloud. 

Will also need to modify logic in __main__.py export_photo to check to see if download-missing flag was used:
https://github.com/RhetTbull/osxphotos/blob/1e013b6802e49e26ec5a94eb702e841b2eb68395/osxphotos/__main__.py#L1163-L1178

Regarding the default, Photos does not export the live photo if using a regular "export 1 photo" feature, only if using "unmodified originals".  I can certainly understand your logic on --export-live default but it's the way it is for 2 reasons: 
1. I added this feature later in the development so it changed the behavior
2. I prefer maximum control so the user knows exactly what's going to happen  -- e.g. export-edited is also not a default

I'll think about the defaults some more.  It would be possible to change this to a --no-export-live for example.  Another option might be a .osxphotos or .ini config file that let you set the defaults you like.
